packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -190,48 +119,48 @@ Future<Null> buildGradleProject(BuildMode buildMode) async {
   settings.values['flutter.buildMode'] = buildModeName;
   settings.writeContents(localProperties);
 
-  String gradlew = await ensureGradlew();
+  String gradle = await ensureGradle();
 
   switch (flutterPluginVersion) {
     case FlutterPluginVersion.none:
       // Fall through. Pretend it's v1, and just go for it.
     case FlutterPluginVersion.v1:
-      return buildGradleProjectV1(gradlew);
+      return buildGradleProjectV1(gradle);
     case FlutterPluginVersion.managed:
       // Fall through. Managed plugin builds the same way as plugin v2.
     case FlutterPluginVersion.v2:
-      return buildGradleProjectV2(gradlew, buildModeName);
+      return buildGradleProjectV2(gradle, buildModeName);
   }
 }
 
 String _escapePath(String path) => platform.isWindows ? path.replaceAll('\\', '\\\\') : path;
 
-Future<Null> buildGradleProjectV1(String gradlew) async {
-  // Run 'gradlew build'.
-  Status status = logger.startProgress('Running \'gradlew build\'...', expectSlowOperation: true);
+Future<Null> buildGradleProjectV1(String gradle) async {
+  // Run 'gradle build'.
+  Status status = logger.startProgress('Running \'gradle build\'...', expectSlowOperation: true);
```

Comment:
we should maybe check how consistent we are about this, but generally we use `"` quotes for strings in output